CardPayments/src/main/java/com/paypal/android/cardpayments/CardAnalytics.kt

```diff
@@ -3,9 +3,7 @@ package com.paypal.android.cardpayments
 import com.paypal.android.corepayments.analytics.AnalyticsService
 
 @Suppress("TooManyFunctions")
-class CardAnalytics(
-    private val analyticsService: AnalyticsService
-) {
+internal class CardAnalytics(private val analyticsService: AnalyticsService) {
```

Comment:
Marking analytics related classes `internal`. I forgot to add this in the previous PR so adding it here now.